publish-hashnode/src/index.ts

```diff
@@ -11,40 +11,48 @@ const main = async () => {
         const filePath = path.join(__dirname, "../../", file)
 
         const {frontMatter, content} = extractFrontMatterAndContent(filePath)
-        const {seo_title, seo_description, article_title, article_subtitle, slug, canonical_url} = frontMatter
+        const {seo_title, seo_description, title, subtitle, slug, canonical_url, cover_image, coAuthors} = frontMatter
```

Comment:
```suggestion
        const {seo_title, description, title, subtitle, slug, canonical_url, image, authors} = frontMatter
```
seo_title is optional if not present title should be used

Reply:
seo_description -> description